checksum.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+RESULT_FILE=$1
+
+if [[ -f ${RESULT_FILE} ]]; then
+  rm ${RESULT_FILE}
+fi
+touch ${RESULT_FILE}
+
+checksum_file() {
+  echo $(openssl md5 $1 | awk '{print $2}')
+}
+
+FILES=()
+while read -r -d ''; do
+	FILES+=("$REPLY")
+done < <(find . -type f \( -name "build.gradle*" -o -name "settings.gradle*" -o -name "gradle-wrapper.properties" \) -print0)
```

Comment:
I see that the example on the cache repo uses `hashFiles(**/*.gradle)` (https://github.com/actions/cache/blob/master/examples.md#java---gradle) -- would that work for us here instead of manually MD5ing, or do we need `gradle-wrapper.properties` too? I guess we might pull the directory and not use it if gradle-wrapper.properties has changed...

Reply:
Talked offline - more flexible to just maintain a script to point to